### PR TITLE
Bump all the things

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: ember-addon-main.js [hunk 1]
```diff
@@ -18,26 +18,26 @@ module.exports = {
 
   parentRegistry: null,
 
-  shouldSetupRegistryInIncluded: function() {
+  shouldSetupRegistryInIncluded() {
```
Comment (rwjblue) [inline ember-addon-main.js]: This should go, we should remove support for old ember-cli during the major bump

File: ember-addon-main.js [hunk 2]
```diff
@@ -47,30 +47,30 @@ module.exports = {
     }
   },
 
-  included: function (app) {
+  included (app) {
     this._super.included.apply(this, arguments);
 
     if (this.shouldSetupRegistryInIncluded()) {
       this.setupPreprocessorRegistry('parent', app.registry);
```
Comment (rwjblue) [inline ember-addon-main.js]: 🙈

File: ember-addon-main.js [hunk 3]
```diff
@@ -116,30 +116,30 @@ module.exports = {
...
+          let pluginHashForDep = hashForDep(wrapper.baseDir());
           cacheKeys.push(pluginHashForDep);
         }
         if (augmentsCacheKey) {
           cacheKeys.push(wrapper.cacheKey());
         }
       } else {
         // support for ember-cli < 2.2.0
-        var log = this.ui.writeDeprecateLine || this.ui.writeLine;
+        let log = this.ui.writeDeprecateLine || this.ui.writeLine;
```
Comment (rwjblue) [inline ember-addon-main.js]: Can remove the fallback here.

File: ember-addon-main.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+const hashForDep = require('hash-for-dep');
 
 module.exports = {
   name: 'ember-cli-htmlbars',
 
-  init: function() {
+  init() {
     if (this._super.init) { this._super.init.apply(this, arguments); }
 
     let checker = new VersionChecker(this);
```
Comment (rwjblue) [inline ember-addon-main.js]: Is this version checker still needed? Can we just require > ember-cli@2.2?

**Comment by stefanpenner** [comment]
released as v2.0.0 🎉

**Pull request merged**